projects/plugins/protect/src/js/components/admin-page/index.jsx

```diff
@@ -47,6 +57,35 @@ const Admin = () => {
 		);
 	}
 
+	/*
+	 * When there's no information yet. Usually when the plugin was just activated
+	 */
+
+	if ( ! lastChecked ) {
+		return (
+			<AdminPage moduleName={ __( 'Jetpack Protect', 'jetpack-protect' ) }>
+				<AdminSectionHero>
+					<Container horizontalSpacing={ 3 } horizontalGap={ 7 }>
+						<Col sm={ 4 } md={ 4 } lg={ 6 }>
+							<H3 mt={ 8 }>{ __( 'Your results will be ready soon', 'jetpack-protect' ) }</H3>
```

Comment:
I'm not sure about adding the mt here. I think it should be addressed at layout level

Reply:
Known issue: See note 1 in the test instructions :)